### DISCUSSION TITLE
Support <script is:inline type="text/wxtag-template"></script>

### DISCUSSION THREAD

**Discussion by yoyo837** [withastro/roadmap#865] [Proposal]
### Body

# Summary
Support `<script is:inline type="text/wxtag-template"></script>`
# Background & Motivation

In China, WeChat Mini Program is a software community with a large number of users. It uses
Code like this renders content:

```html
    <wx-open-launch-weapp
      appid="wx0321a450a049a3db"
      path="views/pages/index/index"
    >
      <script type="text/wxtag-template">
        <style >
         .btn { padding: 12px }
         style here
         </style>
        <div>
          layout here
         </div>
      </script>
    </wx-open-launch-weapp>
```

# Goals

Link the huge WeChat mini program ecosystem

# Example

**Comment by florian-lefebvre** [comment]
What happens currently if you do this?
